src/index.ts

```diff
@@ -31,10 +31,15 @@ export {
   DealerAnalytics,
   BuyNowConfiguration,
   MessageLeadListing,
+  EmploymentType,
+  Gender,
```

Comment:
Do we ask for gender in any of the forms in listings? IIRC it was used in MBG, but that's gone, but online purchase or buyer protection? If yes let's align listings and use exposed enum there.

Reply:
> but online purchase or buyer protection?

Checked, neither use gender in any form